src/js/themes/hpe.js

```diff
@@ -874,141 +894,160 @@ export const hpe = deepFreeze({
   },
   heading: {
     color: 'text-strong',
+    weight: 450,
     level: {
       1: {
         small: {
-          size: '36px',
-          height: '42px',
+          size: '48px', // size: '36px',
+          height: '48px', // height: '42px',
```

Comment:
I'm wondering if we should adjust the theme to allow setting `heading.height: '1em'` generally, which would then apply to all levels and weights.